card/mod/05_standard/set/self/signin.rb

```diff
@@ -96,6 +95,10 @@ def default_core_args args={}
   end  
 end
 
+event :signin_success, :after=>:signin do
+  abort :success
+end
+
```

Comment:
Wondering if you would use :after => :signin of :before => :signin_success to add something after signin?  As long as it is loaded after this code, I think either works, but is one better?